buildSrc/src/main/kotlin/whitefox.java-conventions.gradle.kts

```diff
@@ -33,7 +33,7 @@ tasks.withType<JavaCompile>().configureEach {
 
 tasks.register("devCheck") {
     dependsOn(tasks.spotlessApply)
-    finalizedBy(tasks.check)
+    finalizedBy(tasks.test)
```

Comment:
why this has changed?

Reply:
because I did not have the correct env values at the time for IT tests 😄 Ill revert it back when I fix other comments